libs/common/src/utils/date.ts

```diff
@@ -16,6 +16,15 @@ export const isValidIsoMonth = (value: unknown): value is IsoMonthStamp => isStr
 
 export const toIsoMonthStamp = (date: Date): IsoMonthStamp => format(date, 'yyyy-MM');
 
+/**
+ * generates string of month and year from MonthAndYear object
+ * eg. November 2024 from { month: 11, year: 2024 }
+ */
+export const toMonthYearString = ({ month, year }: MonthAndYear): IsoMonthStamp => {
+  const date = new Date(year, month - 1);
+  return format(date, 'MMMM yyyy');
+};
```

Comment:
We shouldn't need this, we have formatters for report period which should be being used to format report periods to ensure we are consistently supporting quarterly reports.

Reply:
removed